ddtrace/opentrace/settings.py

```diff
@@ -0,0 +1,45 @@
+import opentracing
+
+from ddtrace import Pin
+from ddtrace.ext import AppTypes
+from ddtrace.settings import ConfigException
+
+from .tracer import Tracer
+
+
+class Config(object):
```

Comment:
I would say to remove this implementation for this PR. Let's keep the skeleton an empty placeholder for everything we need. Then let's open a new PR where I can leave comments only for the `Config` implementation. Sounds good?